## Introduction

Welcome to the data repository for the *Solenodon paradoxus* genome. See below for descriptions how to retrieve appropriate files in standard formats from public repositories.

## Publication

The data described below was published in the following paper. Please cite this publication when using these data.

Casewell, N.R., D. Petras, D.C. Card, V. Suranse, A.M. Mychajliw, D. Richards, I. Koludarov, L.-O. Albulescu, J. Slagboom, B.-F. Hempel, N.M. Ngum, R.J Kennerley, J.L. Brocca, G. Whiteley, R.A. Harrison, F.M.S. Bolton, J. Debono, F.J. Vonk, J. Alföldi, J. Johnson, E. Karlsson, K. Lindblad-Toh, I. Mellor, R.D. Süssmuth, B.G. Fry, S. Kuruppu, W.C. Hodgson, J. Kool, T.A. Castoe, I. Barnes, K. Sunagar, E.A.B. Undheim, & S.T. Turvey. *In Prep.* Solenodon genome reveals convergent evolution of venom in eulipotyphlan mammals.

## Raw Sequencing Data

The raw sequence read data for this genome can be obtained from the NCBI Sequence Read Archive under accession [XXX](FIXME).

## Genome Assemblies

The raw genome assembly prior to NCBI submission (FASTA format) can be found at Figshare under [accession doi 10.6084/m9.figshare.7640456](https://doi.org/10.6084/m9.figshare.7640456).

The genome assembly (FASTA format) has also been accessioned through NCBI under accession [RJWH00000000](FIXME).

Note: NCBI filters away contigs and scaffolds based on certain criteria, so differences in the assemblies will exist. Moreover, the genome annotation described in the above publication (see below) is based on the pre-NCBI genome assembly.

## Genome Annotation

Repeat and gene annotation files for the raw (pre-NCBI) genome assembly (GFF and FASTA files), which are described in the above publication, can be found at Figshare under [accession doi 10.6084/m9.figshare.7640456](https://doi.org/10.6084/m9.figshare.7640456). 

We have also created a [Genome Hub](https://genome.ucsc.edu/goldenpath/help/hgTrackHubHelp.html) that can be accessed through the [UCSC Genome Browser](https://genome.ucsc.edu/cgi-bin/hgHubConnect), followed by clicking on the “My Hubs” tab and then copying the URL [https://de.cyverse.org/anon-files/iplant/home/darencard/solPar1_hub/assembly_hub.hub.txt](https://de.cyverse.org/anon-files/iplant/home/darencard/solPar1_hub/assembly_hub.hub.txt) and clicking “Add Hub”. 

NCBI also generally performs gene annotation via their [Eukaryotic Genome Annotation Pipeline](https://www.ncbi.nlm.nih.gov/genome/annotation_euk/process/) and the resulting annotation can be found under accession [XXX](FIXME).

## Proteomic Data

Mass spectrometry proteomics data (.mgf, .raw, .mzXML and PSM/ PrSM output files as well as deconvoluted spectra) have been deposited to [ProteomeXchange](http://proteomecentral.proteomexchange.org) with the accession [PXD009593](http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD009593) via the [Mass Spectrometry Interactive Virtual Environment (MassIVE)](https://massive.ucsd.edu/) with the accession [MSV000082307](https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=04342867c1304a0dae67b886cce42605).
